libs/platform/src/lib/components/form/form-generator/form-generator.service.ts

```diff
@@ -0,0 +1,362 @@
+import { Injectable, Type } from '@angular/core';
+import { AsyncValidatorFn, FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Observable } from 'rxjs';
+
+import { DynamicFormControl } from './dynamic-form-control';
+import { DynamicFormItem, DynamicFormItemChoices } from './interfaces/dynamic-form-item';
+import { FormComponentDefinition } from './interfaces/form-component-definition';
+import { DEFAULT_COMPONENTS_LIST } from './config/default-components-list';
+import { DEFAULT_VALIDATION_ERRORS } from './config/default-validation-errors';
+import { BaseDynamicFormGeneratorControl, SelectItem } from '@fundamental-ngx/platform';
```

Comment:
please use the appropriate imports path

Reply:
Platform does not support deep imports yet